src/common/utility_win.cpp

```diff
@@ -346,6 +347,17 @@ QString Utility::formatWinError(long errorCode)
     return QStringLiteral("WindowsError: %1: %2").arg(QString::number(errorCode, 16), QString::fromWCharArray(_com_error(errorCode).ErrorMessage()));
 }
 
+QString Utility::getCurrentUserName()
+{
+    TCHAR username[UNLEN + 1] = {0};
+    DWORD len = sizeof(username) / sizeof(TCHAR);
+    
+    if (!GetUserName(username, &len)) {
+        qCWarning(lcUtility) << "Could not retrieve Windows user name." << formatWinError(GetLastError());
```

Comment:
@mgallien https://docs.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-getusernamea does not say anything about that.

I have performed a simple test. I've changed the size of the buffer to `2` to make sure the call to `GetUserName` fails. The buffer remained empty after the `GetUserName` failure. Is it enough prove to keep it like that, or, should I return `{}` in case of `GetUserName` failure?

Reply:
OK, thanks for the tests
I would expect the doc to not always tell you about what happen in error cases